README.md

```diff
@@ -56,10 +58,17 @@ Please check the [roadmap](https://github.com/AzureAD/microsoft-authentication-l
 
 - [User Voice page](https://feedback.azure.com/forums/169401-azure-active-directory) to provide recommendations and/or feedback.
 
+## Share Your Experience
+How easy was it to use this library? And valuable did you find it? Annonymously share your experience with us [here](https://ncv.microsoft.com/NhQGJ4UuqI).
+
 ## Contribute
 
 We enthusiastically welcome contributions and feedback. Please read the [contributing guide](contributing.md) before you begin.
 
+## Share your experience
+
+How easy was it to use this library? And valuable did you find it? Anonymously share your experience with us [here](https://ncv.microsoft.com/NhQGJ4UuqI).
```

Comment:
Looks like this section was added twice to this file.

Reply:
Thanks Jason, updating in new revision.